deps/build.jl

```diff
@@ -16,6 +16,55 @@ function library_name(v)
     return "$(cpx_prefix)cplex$(v).$(Libdl.dlext)"
 end
 
+function possible_paths(cplex_studio_paths::Vector{String})
```

Comment:
Since we only ever pass a single string, you probably just want this to be 
`function possible_path(cplex_studio_version::String)`? 

Then you can remove the map in favor of just the `if`.